CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ deprecated, and replaced by `storage`.
 - When providing a storage that already contains session information in the NodeJS
 context (as demonstrated with the `FileSystemStorage` in `packages/node/example/bootstrappedApp`),
 the refresh token is now picked up instead of the request failing.
+- Calling `Session::fetch` before logging the Session in threw an error.
```

Comment:
Note: this changelog is shared between all published packages, so probably a good idea to split the notes by affected package too.